.travis.yml

```diff
@@ -22,6 +22,11 @@ script: ./vendor/bin/phpunit
 
 jobs:
   include:
+    - stage: Test
+      php: 7.1
+      install:
+        - travis_retry composer -n --prefer-source --prefer-lowest update
```

Comment:
why using `--prefer-source` (while persisting the composer cache in the Travis cache at the same time) ? Using `--prefer-dist` will benefit from the persisted cache.

Reply:
BTW: Using `--no-interaction` is more descriptive than just `-n` and makes the travis-file easier to read.

Reply:
Good point. I’ll update this 👍